.github/workflows/check-e2e.yml

```
@@ -54,10 +57,20 @@ jobs:
         if: steps.playwright-dep-cache.outputs.cache-hit != 'true'
         run: npx playwright install chromium firefox
 
+      - id: auth
+        uses: google-github-actions/auth@v1
+        with:
+          workload_identity_provider: "projects/281042598092/locations/global/workloadIdentityPools/github-actions/providers/google-cloud"
+          service_account: "github-actions@radicle-services.iam.gserviceaccount.com"
+
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v1
+
       - name: Start http-api test server
         run: |
           if [ ${{ matrix.browser }} = "heartwood" ]; then
-            ./scripts/run-httpd-with-fixtures --non-interactive --detach
+            export PATH=".:$PATH"
+            ./scripts/run-httpd-with-fixtures --non-interactive --binary &
```

Should we try to collecting the logs?

```suggestion
./scripts/run-httpd-with-fixtures --non-interactive --binary >> tests/artifacts/httpd-${{ matrix.browser }}.log 2>&1 &
```